geotrek/feedback/views.py

```diff
@@ -27,6 +26,33 @@ class ReportLayer(mapentity_views.MapEntityLayer):
     filterform = ReportFilterSet
     properties = ["email"]
 
+    def get_queryset(self):
+        qs = feedback_models.Report.objects.existing().select_related(
```

Comment:
best practice is to use super().get_queryset. In this case dont repeat defined queryset by using self.queryset